### DISCUSSION TITLE
No support for per-website autofill?

### DISCUSSION THREAD

**Discussion by worldofgeese** [zeapo/Android-Password-Store#1885] [Q&A]
Password Store appears to view Firefox as a single URI so passwords I've saved across existing websites don't suggest, only passwords created inside Firefox with the Password Store popup autofill. I am using the latest 2.0 snapshot to get around a bug affecting GrapheneOS users.

**Comment by msfjarvis** [comment]
Firefox is supported for full Autofill and should work, unless you're using some unofficial homebrew build that either isn't signed by the official keys or has a different package name. Do either of those apply to your currently installed Firefox build?

**Comment by worldofgeese** [comment]
Ah, that must be it. I'm using Mull, a privacy fork of Firefox for Android.

**Reply by msfjarvis** [reply]
Support for Mull was [added a few months ago](https://github.com/android-password-store/Android-Password-Store/pull/1640) and is available in the [snapshot builds](https://docs.passwordstore.app/docs/users/release-channels).

**Comment by worldofgeese** [comment]
In that case, it isn't working for me, even with the latest snapshot build as of today. You can see here Password Store thinks all of Mull is a single URI target
![Screenshot_20220501-133211](https://user-images.githubusercontent.com/59834693/166144015-9f0e1b99-74a5-4ec7-a04b-75b8fb344493.png)

The carta suggestion is for the Carta website yet it is the first choice for every website login.

**Reply by msfjarvis** [reply]
That's odd, I'll investigate it.
